lib/active_model/serializer/railtie.rb

```diff
@@ -19,5 +19,7 @@ class Railtie < Rails::Railtie
       app.load_generators
       require 'generators/serializer/resource_override'
     end
+
+    ActionController::TestCase.send(:include, ActiveModelSerializers::Test::Schema)
```

Comment:
I don't think we need this `if Rails.env.test?` Active Job and Action Mailer for example does not do that as you can see [here](https://github.com/rails/rails/blob/d7add5352e7c8b31850431bb93acb56d93c5dc64/activejob/lib/active_job.rb#L34), [here](https://github.com/rails/rails/blob/d7add5352e7c8b31850431bb93acb56d93c5dc64/activejob/lib/active_job/test_case.rb), [here](https://github.com/rails/rails/blob/d7add5352e7c8b31850431bb93acb56d93c5dc64/actionmailer/lib/action_mailer.rb#L46-L47) and [here](https://github.com/rails/rails/blob/d7add5352e7c8b31850431bb93acb56d93c5dc64/actionmailer/lib/action_mailer/test_helper.rb).